examples/fluids/qfunctions/newtonian.h

```diff
@@ -231,8 +230,10 @@ CEED_QFUNCTION_HELPER int IFunction_Newtonian(void *ctx, CeedInt Q, const CeedSc
 
     for (CeedInt j = 0; j < 5; j++) v[j][i] = wdetJ * (U_dot[j] - body_force[j]);
     if (context->idl_enable) {
+      const CeedScalar sigma   = LinearRampCoefficient(context->idl_amplitude, context->idl_length, context->idl_start, x_i[0]);
+      jac_data[14 * Q + i]     = sigma;
```

Comment:
```suggestion
       StoredValuesPack(Q, i, 14, 1, sigma, jac_data);
```

A bit overkill, but I'd like to storage and access it consistent.